packages/twenty-front/src/modules/auth/sign-in-up/components/SignInUpForm.tsx

```diff
@@ -76,6 +90,39 @@ export const SignInUpForm = () => {
     }
   };
 
+  const handleResetPasswordClick = async () => {
```

Comment:
let's move this to a dedicated hook in modules/auth/sign-in-up/hooks
We already have a useSignInUp that could host this function hook but it's time to break it into pieces. could you also break useSignInUp into pieces?

Reply:
@charlesBochet, I can't see how handleResetPasswordClick fits into a hook. Could you please explain a little bit more, what we are trying to achieve here? 
Also, `useSignInUp` should be broken into pieces, please explain how it should be broken as `useSignInUP` has already named exports. I think, the way different pieces are exported to utilize specifically but all relevant code for signinup is placed in same hook seems to be better structured. Let me know how this should be architectured instead.